crates/sncast/tests/helpers/constants.rs

```diff
@@ -18,3 +20,15 @@ pub const DEVNET_PREDEPLOYED_ACCOUNT_ADDRESS: &str =
 // https://github.com/0xSpaceShard/starknet-devnet-rs/blob/1a76e9d29541af2667ca815c47bb332cead27c55/crates/starknet/src/constants.rs#L17
 pub const DEVNET_OZ_CLASS_HASH_CAIRO_1: &str =
     "0x61dac032f228abef9c6626f995015233097ae253a7f72d68552db02f2971b8f";
+
+pub const MAP_CONTRACT_ADDRESS_SEPOLIA: &str =
+    "0xcd8f9ab31324bb93251837e4efb4223ee195454f6304fcfcb277e277653008";
```

Comment:
I wonder if it wouldn't be better to use forked class hashes but deploy contract instance per-test. Would prevent race conditions.

Reply:
Generally agree that ideally it should be done this way. However, given that it has been done in a similar manner so far, and that tests don't depend on each other (the order doesn't matter) - I am not sure that making this change would be beneficial.

Reply:
Just a suggestion, ok with leaving it as it is.